lib/peep.rb

```diff
@@ -0,0 +1,46 @@
+require "pg"
+require "time"
+
```

Comment:
Did you consider doing something like this, to build hashes with all the details?

```ruby
peeps.map do |peep|
   { :message => peep["message"] , date: peep['date'], time: peep['time']}
end
```